web/src/app/services/ServiceDetails.js

```diff
@@ -117,21 +102,47 @@ export default function ServiceDetails({ serviceID }) {
             )}
           </div>
         }
+        pageContent={<ServiceOnCallList serviceID={serviceID} />}
+        secondaryActions={[
+          {
+            label: 'Edit',
+            icon: <Edit />,
+            handleOnClick: () => setShowEdit(true),
+          },
+          {
+            label: 'Delete',
+            icon: <Delete />,
+            handleOnClick: () => setShowDelete(true),
+          },
+          <QuerySetFavoriteButton
+            key='secondary-action-favorite'
+            serviceID={serviceID}
+          />,
+        ]}
         links={[
           {
             label: 'Alerts',
             status: alertStatus(_.get(data, 'alerts.nodes')),
             url: 'alerts',
+            subText: 'All alerts specific to this service',
```

Comment:
I feel like it may seem verbose to us since we work with these bits so often, but I think this type of information could go a long way- for example, we often get questions from customers on how to do simple things like find a rotation for a schedule, edit an escalation policy, what labels are, etc. As of right now, most of those pages are flat lists without any contextual information. Curious of others thoughts as well